pom.xml

```
@@ -31,7 +31,7 @@
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <maven.javadoc.skip>true</maven.javadoc.skip>
-        <jenkins.version>2.303.3</jenkins.version>
+        <jenkins.version>2.346</jenkins.version>
```

I appreciate the upgrade, but I'd prefer to keep a LTS here.

```suggestion
        <jenkins.version>2.346.1</jenkins.version>
```

---

Thanks.